src/routes.js

```diff
@@ -122,6 +124,13 @@ module.exports = [
     }
   },
   {
+    method: 'POST',
+    path: '/finished',
+    handler: (request, reply) => {
+      reply(request.payload.test ? '' : request.payload);
```

Comment:
I was doing this because I was sending an email ever time I ran the tests. What you have said makes sense.